packages/dashboard/src/features/database/components/TableForm.tsx

```
@@ -29,6 +31,212 @@ const newColumn: TableFormColumnSchema = {
+
+  if (schemaName && draft.schemaName && draft.schemaName !== schemaName) {
+    return false;
+  }
+
+  return hasCreateDraftData(draft);
+};
+
+export const clearTableFormCreateDraft = (draftScope?: string, schemaName?: string) => {
+  getTableFormDraftStorage()?.removeItem(
+    getTableFormCreateDraftStorageKey(draftScope, schemaName)
+  );
```

ESLint

[error] Replace `⏎····getTableFormCreateDraftStorageKey(draftScope,·schemaName)⏎··` with `getTableFormCreateDraftStorageKey(draftScope,·schemaName)`

(prettier/prettier)